hw/ip/otbn/util/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -e
+
+if [ "$#" -ne 2 ]; then
+  echo "Basic script to build a single .S asm file for OTBN. Input file is first"
```

Comment:
Nit: Probably "to assemble a single OTBN file to a binary" or something. At the moment, it sounds like the .S asm file is the output.